tests/unittest/data/smallpark/readme.txt

```diff
@@ -0,0 +1,20 @@
+Author
+======
+
+This is the work of Emil Persson, aka Humus.
+http://www.humus.name
+humus@comhem.se
+
+
+
+Legal stuff
+===========
+
+This work is free and may be used by anyone for any purpose
+and may be distributed freely to anyone using any distribution
+media or distribution method as long as this file is included.
+Distribution without this file is allowed if it's distributed
+with free non-commercial software; however, fair credit of the
+original author is expected.
+Any commercial distribution of this software requires the written
+approval of Emil Persson.
```

Comment:
Not sure where these files (smallpark/*) came from, but on the referenced website I find only the crossed version.
http://www.humus.name/index.php?page=Textures&start=128
Park 2 http://www.humus.name/Textures/Park2.zip

and the legal stuff is a bit different:

Author
======

This is the work of Emil Persson, aka Humus.
http://www.humus.name



License
=======

This work is licensed under a Creative Commons Attribution 3.0 Unported License.
http://creativecommons.org/licenses/by/3.0/`

Reply:
Maybe add ref for the two other enmap files. Do we really need the >30Mo pfm file ?

Reply:
> Not sure where these files (smallpark/*) came from, but on the referenced website I find only the crossed version. http://www.humus.name/index.php?page=Textures&start=128 
> Park 2 http://www.humus.name/Textures/Park2.zip

If you download and uncompress Park2.zip (I did it today to check), you will have the textures per-face, not the displayed cross version. In smallpark, I resized the textures down  to take less memory. I dowloaded these files several years ago (06-20-2012) and the licence file is the one from my download, not the one from the actual distributed file.

Reply:
> Maybe add ref for the two other enmap files. Do we really need the >30Mo pfm file ?

I'll add the ref in a few minutes. For the PFM, if you have a lighter version, you can replace it. 
These images allow to cover the test for all the supported parameterizations.